lib/class-wp-rest-image-editor-controller.php

```diff
@@ -116,6 +116,72 @@ public function register_routes() {
 				),
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/apply',
+			array(
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'apply_edits' ),
+					'permission_callback' => array( $this, 'permission_callback' ),
+					'args'                => array(
+						array(
```

Comment:
This API is confusing, but when specifying `args` it already assumes that the overall type is an object, and the `args` array is a collection of property names => schemas.

So we should drop the overall `[ type => 'object', 'properties' => [] ]` and instead have it be `args.crop => []` etc...

Reply:
Does doing that make the property required?

Reply:
No, properties aren't required by default, only if `required` is set for that property name.

Not required
```php
$args = [
  'crop' => [
      'type' => 'object',
     'properties' => [],
   ]
];
```

Required
```php
$args = [
  'crop' => [
      'type' => 'object',
      'required' => true,
      'properties' => [],
   ]
];
```